src/westpa/analysis/core.py

```diff
@@ -11,12 +11,12 @@
 
 
 class Run:
-    """A completed WESTPA simulation run.
+    """A read-only view of a WESTPA simulation run.
 
     Parameters
     ----------
     h5filename : str or file-like object, default 'west.h5'
-        Pathname or stream of a WESTPA HDF5 file.
+        Pathname or stream of a WEST HDF5 file.
```

Comment:
Just to be in line with the terminology of the WESTPA 2.0 paper. 

[...] of a main WESTPA HDF5 data file.

Reply:
Oops, I was trying to be in line with https://westpa.github.io/westpa/users_guide/hdf5.html. I'll change this to be consistent with the paper.

Reply:
> Oops, I was trying to be in line with https://westpa.github.io/westpa/users_guide/hdf5.html. I'll change this to be consistent with the paper.

No worries. That was written when WESTPA was still called 'west' and had no concept of per-iteration h5s, so it's a bit outdated.